### PR TITLE
Profit interface

### PR DESCRIPTION
**Pull request by loct2205**

### CODE REVIEW

File: src/main/java/com/mycompany/mediaone/View/Profit/ProfitInterface.java [hunk 1]
```diff
@@ -674,7 +632,45 @@ private void btnloadBillActionPerformed(java.awt.event.ActionEvent evt) {//GEN-F
     }//GEN-LAST:event_btnloadBillActionPerformed
 
     private void btnloadBillMouseClicked(java.awt.event.MouseEvent evt) {//GEN-FIRST:event_btnloadBillMouseClicked
+        System.out.println("Hello list out");
```
Comment (lehaithanh1510) [inline src/main/java/com/mycompany/mediaone/View/Profit/ProfitInterface.java:635]: remove system.out.println

File: src/main/java/com/mycompany/mediaone/View/Profit/ProfitInterface.java [hunk 2]
```diff
@@ -14,31 +17,31 @@
...
 import java.util.UUID;
+import java.util.stream.Collectors;
 import javax.swing.JOptionPane;
 import javax.swing.table.DefaultTableModel;
 
 public class ProfitInterface extends javax.swing.JPanel {
 
     private List<Bill> listSoldBill = new ArrayList<>();
     private List<Bill> listBuyBill = new ArrayList<>();
+    //private List<Bill> listBills = this.homePage.billInterface.listBills;
```
Comment (lehaithanh1510) [inline src/main/java/com/mycompany/mediaone/View/Profit/ProfitInterface.java:28]: remove this

File: src/main/java/com/mycompany/mediaone/View/Profit/ProfitInterface.java [hunk 3]
```diff
@@ -674,7 +632,45 @@ private void btnloadBillActionPerformed(java.awt.event.ActionEvent evt) {//GEN-F
...
+        listBuyBill = this.homePage.billInterface.listBills.stream()
+                    .filter(bill -> "buy".equals(bill.getType()))
+                    .collect(Collectors.toList());
+        
+        
+        listSoldBill = this.homePage.billInterface.listBills.stream()
+                    .filter(bill -> "sold".equals(bill.getType()))
+                    .collect(Collectors.toList());
+        
+        //System.out.println("size list: " + listSoldBill.size());
```
Comment (lehaithanh1510) [inline src/main/java/com/mycompany/mediaone/View/Profit/ProfitInterface.java:645]: remove unused code

File: src/main/java/com/mycompany/mediaone/View/Profit/ProfitInterface.java [hunk 4]
```diff
@@ -674,7 +632,45 @@ private void btnloadBillActionPerformed(java.awt.event.ActionEvent evt) {//GEN-F
...
+        System.out.println("size list: " + this.homePage.billInterface.listBills.size());
```
Comment (lehaithanh1510) [inline src/main/java/com/mycompany/mediaone/View/Profit/ProfitInterface.java:646]: remove system.out.println

File: src/main/java/com/mycompany/mediaone/View/Profit/ProfitInterface.java [hunk 5]
```diff
@@ -6,6 +6,9 @@
 import com.mycompany.mediaone.Model.BillModel.BillSold;
 import com.mycompany.mediaone.Model.BillModel.Customer;
 import com.mycompany.mediaone.View.HomePage;
+import com.mycompany.mediaone.View.Bill.BillInterface;
+import com.mycompany.mediaone.View.Bill.BillSoldDetail;
+import com.mycompany.mediaone.View.Bill.BillBuyDetail;
```
Comment (dungvv-qsoft) [inline src/main/java/com/mycompany/mediaone/View/Profit/ProfitInterface.java:11]: Remove unused import

Comment (lehaithanh1510) [inline src/main/java/com/mycompany/mediaone/View/Profit/ProfitInterface.java:11]: remove unused import

**Comment by lehaithanh1510** [comment]
use real data rom list bill to render table in profit interface

**Pull request merged**